src/projection/incflo_projection_bc.cpp

```diff
@@ -23,7 +23,11 @@ incflo::get_projection_bc (Orientation::Side side) const noexcept
             case BC::direction_dependent:
             case BC::mixed:
             {
-                r[dir] = LinOpBCType::inflow;
+                if ( m_use_cc_proj ) {
+                    r[dir] = LinOpBCType::Neumann;
```

Comment:
@asalmgren LinOpBCType::inflow is only currently defined for NodeLap, so this was crashing for the cc_proj. I think this will suffice for the cc_proj.